src/components/SimplePriceChart/SimplePriceChart.tsx

```diff
@@ -56,11 +61,11 @@ const MarketDataChart: React.FC<SimplePriceChartProps> = ({
   useEffect(() => {
     setTimeout(() => {
       if (durationSelector === 0) {
-        setPrice(data?.slice((data?.length * 29) / 30))
+        setPrice(dayData?.slice(-24))
```

Comment:
why -24? Is this taking the last 24 days as default?

Reply:
The first 2 days are pulled in hourly here, so -24 is indicating the last 24 hours. This can definitely be extended to include more days, and I'll match it to the chart that currently exists. Perhaps hourly for the first month, and beyond is daily? 

This sets hourly for most recent 48 hours, then daily beyond that.

Reply:
Got it makes sense. Lets see what hourly for 48 hours + daily beyond that looks like. It sounds like it should be fine.